fidesctl/src/fidesctl/cli/__init__.py

```diff
@@ -78,6 +78,32 @@ def cli(ctx: click.Context, config_path: str, local: bool) -> None:
     if not ctx.invoked_subcommand:
         click.echo(cli.get_help(ctx))
 
+    if (
+        ctx.obj["CONFIG"].user.analytics_opt_out is None
+        and ctx.invoked_subcommand.lower() != "init"
+    ):
+        ctx.obj["CONFIG"].user.analytics_opt_out = bool(
+            input(OPT_OUT_COPY).lower() == "n"
+        )
+        ctx.obj["CONFIG"].cli.analytics_id = ctx.obj[
+            "CONFIG"
+        ].cli.analytics_id or generate_client_id(FIDESCTL_API)
+
+        config_updates = {
+            "cli": {"analytics_id": ctx.obj["CONFIG"].cli.analytics_id},
+            "user": {"analytics_opt_out": ctx.obj["CONFIG"].user.analytics_opt_out},
+        }
+
+        try:
+            update_config_file(config_updates)
+        except FileNotFoundError as err:
+            echo_red(f"Failed to update config file: {err.strerror}")
```

Comment:
Overall this looks to be so much cleaner without the added jumping around, great improvement!